modules/generate-network-policies-ng20.adoc

```diff
@@ -8,14 +8,23 @@
 
 .Procedure
 . In the {product-title-short} portal, navigate to *Network Graph*.
 . Select a cluster, and then select one or more namespaces.
-. In the network graph header, select *Simulate network policy*. {product-title-short} generates policies for all of the deployments that exist in the selected cluster.
-. Optional: In the information panel that opens, select *Exclude ports & protocols* if you do not want ports and protocols to be scoped in {product-title-short} generated policies.
-. Select *Generate and simulate network policies*.
-The generated network policy configuration YAML file opens in the same panel, and the network graph shows the effects of the policies.
+. Optional: Select individual deployments to restrict the policy generated to just those deployments. You can also use the *Filter deployments* feature to further narrow the scope.
+. In the network graph header, select *Network policy generator*.
+. Optional: In the information panel that opens, select *Exclude ports & protocols* to remove the port/protocol restrictions when generating network policies from a baseline. For example, the `nginx3` deployment makes a port 80 connection to `nginx4`, and this is included as part of the baseline for nginx4. If policies are generated and this checkbox is not selected (the default behavior), the generated policy will restrict the allowed connections from `nginx3` to `nginx4` to only port 80. If policies are generated with this option selected, the generated policy will allow any port in the connection from `nginx3` to `nginx4`.
+. Click *Generate and simulate network policies*. {product-title-short} generates policies for the scope that you have chosen. This scope is displayed at the top of the *Generate network policies* panel. Clicking on the deployment information in the scope displays a list of the deployments that are included.
+. The generated network policy configuration YAML file opens in the same panel. You can copy the displayed YAML file to the clipboard or download it by clicking the download icon in the panel.
+. Optional: To compare the generated network policies to the existing network policies, click *Compare*. The YAML files for existing and generated network policies are shown in a side-by-side view.
++
+[NOTE]
+====
+Some items do not have generated policies, such as namespaces with existing ingress policies or deployments in certain protected namespaces such as as `stackrox` or `acs`.
+====
+. Optional: Click the *Actions* menu to perform the following activities:
+* Share the YAML file with notifiers: Sends the YAML file to one of the system notifiers you have configured (for example, Slack, ServiceNow, or an application that uses generic webhooks). These notifiers are configured by navigating to *Platform Configuration* -> *Integrations*. See the documentation in the "Additional resources" section for more information.
```

Comment:
Suggestion: slight rewording to avoid the use of parentheses for general text

```suggestion
* Share the YAML file with notifiers: Sends the YAML file to one of the system notifiers you have configured, for example Slack, ServiceNow, or an application that uses generic webhooks. These notifiers are configured by navigating to *Platform Configuration* -> *Integrations*. See the documentation in the "Additional resources" section for more information.
```